pydantic_ai_slim/pydantic_ai/models/__init__.py

```diff
@@ -49,6 +49,8 @@
     'gemini-1.5-pro',
     'vertexai:gemini-1.5-flash',
     'vertexai:gemini-1.5-pro',
+    'ollama:llama-3.2:latest',
+    'ollama:qwen2.5-coder:latest',
```

Comment:
I think we can add the ~10 most common models we add to `CommonOllamaModelNames` here.